aisera/README.md

```diff
@@ -0,0 +1,94 @@
+      ![my screenshot](images/integration_setup_3.png)
+   5. Click **OK**. 
+4. Create a data source for this integration.
+   1. Go to **Settings** > **Data Sources** > **New Data Source**.
+   2. In the search bar, select **Integration** in the previous step.
+   3. Enter a **Name** and **Schedule** to run the integration.
+   4. Select _Alerts Learning_ from **Functions**.
+      ![my screenshot](images/data_source_setup_1.png)
+      ![my screenshot](images/data_source_setup_2.png)
+5. Run the data source. You can also choose to run this data source based on a schedule.
+6. Once the alerts have been ingested, Aisera AIOps pipelines will run.
+7. AiOps pipelines generate major incidents/clusters based on the category and severity of incidents received.
+Each major incident contains the following main components:
+      - **Title**: Brief info about the major incident 
+      - **Status**: Open/Close  
+      - **Cluster**: Aisera generated cluster that this major incident belongs to. 
+      - **Outage Probability**: Outage probability for this major incident.
+      - **Impacted Business Services**
+      - **Impacted Systems**: Infrastructure component impacted by this major incident.
+      - **Impacted Storage**: Any DB/Storage that was impacted.
+      - **Impacted Users**: Users which are being impacted by this major incident.
+      - **Incidents**: Any Incident from ticketing systems that caused this major incident.
+      - **Alerts**: Datadog alerts/logs that are part of this major incident.
+      - **Change Request**: Any change request which caused this major incident.
+      - **Timeline**: A timeline of the incident/alerts correlation.
+      - **Root Cause**: A root cause generated by the Aisera LLM model.
+      - **Causal Graphs**: Graph of business services.
+      - **Logs and Trace Analysis**
+      - **Major Incident Connectivity Graph**: A visual representation of how users are impacted by specific alerts/services/infrastructure.
+      - **Score over Time**: AMI scores over time.
+      - **Outage Probability over Time**
+      - **Major Incident Correlation Evidence**
+          ![my screenshot](images/aisera-major_incident_detail_1.png)
+          ![my screenshot](images/aisera-major_incident_detail_2.png)
+          ![my screenshot](images/aisera-major_incident_detail_3.png)
+          ![my screenshot](images/aisera-major_incident_detail_4.png)
+
+Once Aisera detects a major incident:
+
+- It notifies the designated persona on the channel of their choice.
+- Aisera creates an incident in Datadog for Datadog Admins to have a visibility of the
+
+Aisera can require approvals before a remediation task can run.
+
+Aisera can also run any remediation task (Any API call/ Shell/Python/Powershell script/ Terraform manifests etc..), leveraging the no-code platform it provides.
+
+## Data Collected
+
+One you run the datasource, Aisera will collect the metrics,alerts and logs  from datadog.
```

Comment:
This tab will only render in-app if you are sending metrics into Datadog. I'd recommend adding this information under the Overview section with a `###` header